apps/pom.xml

```diff
@@ -259,6 +264,29 @@
 				<scope>test</scope>
 			</dependency>
 
+			<dependency>
+				<!-- Used in subsitute of Java XML Bind -->
+				<groupId>jakarta.xml.bind</groupId>
+				<artifactId>jakarta.xml.bind-api</artifactId>
+				<version>2.3.2</version>
+			</dependency>
+			<dependency>
+				<groupId>org.glassfish.jaxb</groupId>
+				<artifactId>jaxb-runtime</artifactId>
+				<version>2.3.1</version>
+			</dependency>
+			<dependency>
+				<!-- Used for annotations in the generated code. -->
+				<groupId>jakarta.annotation</groupId>
+				<artifactId>jakarta.annotation-api</artifactId>
+				<version>1.3.5</version>
+			</dependency>
+			<dependency>
+				<groupId>javax.servlet</groupId>
+				<artifactId>javax.servlet-api</artifactId>
+				<version>4.0.1</version>
+				<scope>provided</scope>
+			</dependency>	
```

Comment:
this one seems like it belongs in dependencyManagement of the bfd-server project. does anyone else use it?